React_Day9/avengers/src/components/BodyComponent.js

```diff
@@ -0,0 +1,42 @@
+import { useEffect, useState } from "react";
+
+import CardComponent from "./CardComponent";
+import SearchBarComponent from "./SearchBarComponent";
+import NoResultFound from "./NoResultFound";
+import fetchService from "../utils/apiCall.js";
+
+function BodyComponent() {
+  const fetchedData = fetchService();
+  const [avengersData, setAvengersData] = useState([]);
+  const [afterFilterData, setAfterFilterData] = useState([]);
+
+  useEffect(() => {
+    fetchedData.then((res) => {
+      const data = [].concat(...res);
+      setAvengersData(data);
+      setAfterFilterData(data);

```

Comment:
curious about this. Let me know if we can connect over this.

Reply:
sure